Azure/sonic-buildimage
# Eanble Vxlan/EVPN for TD3.

#### Why I did it
fix #8371 
support evpn and vxlan
#### How I did it
enable flow mode in bcm config.
#### How to verify it
follow the issue test step
~~~
config vlan add 1991
config loopback add Loopback0
config int ip add Loopback0 10.10.10.10/32
config vxlan add nve1 10.10.10.10
config vxlan map add nve1 1991 1991
~~~
check syslog without error
~~~
Dec 10 05:28:28.330167 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map:2916 SAI Exit brcm_sai_tnl_mp_create_tunnel_map
Dec 10 05:28:28.332062 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map:2871 SAI Enter brcm_sai_tnl_mp_create_tunnel_map
Dec 10 05:28:28.332062 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map:2916 SAI Exit brcm_sai_tnl_mp_create_tunnel_map
Dec 10 05:28:28.333183 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map:2871 SAI Enter brcm_sai_tnl_mp_create_tunnel_map
Dec 10 05:28:28.333183 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map:2916 SAI Exit brcm_sai_tnl_mp_create_tunnel_map
Dec 10 05:28:28.334264 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map:2871 SAI Enter brcm_sai_tnl_mp_create_tunnel_map
Dec 10 05:28:28.334264 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map:2916 SAI Exit brcm_sai_tnl_mp_create_tunnel_map
Dec 10 05:28:28.334782 as7726-32x-2 NOTICE swss#orchagent: :- create_tunnel: create_tunnel:encapmaplist[0]=0x29000000000683
Dec 10 05:28:28.334782 as7726-32x-2 NOTICE swss#orchagent: :- create_tunnel: create_tunnel:encapmaplist[1]=0x29000000000685
Dec 10 05:28:28.335508 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel:3049 SAI Enter brcm_sai_tnl_mp_create_tunnel
Dec 10 05:28:28.335538 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel:3138 Setting peer_mode to 1
Dec 10 05:28:28.335565 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel:3285 Created tunnel id: 3
Dec 10 05:28:28.335587 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel:3313 SAI Exit brcm_sai_tnl_mp_create_tunnel
Dec 10 05:28:28.336619 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_term_table_entry:3871 SAI Enter brcm_sai_tnl_mp_create_tunnel_term_table_entry
Dec 10 05:28:28.336689 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_term_table_entry:4017 SAI Exit brcm_sai_tnl_mp_create_tunnel_term_table_entry
Dec 10 05:28:28.338114 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map_entry:4193 SAI Enter brcm_sai_tnl_mp_create_tunnel_map_entry
Dec 10 05:28:28.338216 as7726-32x-2 INFO syncd#syncd: [none] SAI_API_TUNNEL:brcm_sai_tnl_mp_create_tunnel_map_entry:4334 SAI Exit brcm_sai_tnl_mp_create_tunnel_map_entry
Dec 10 05:28:28.338767 as7726-32x-2 NOTICE swss#orchagent: :- addOperation: Vxlan tunnel map entry 'map_1991_Vlan1991' for tunnel 'nve1' was created
Dec 10 05:28:28.351010 as7726-32x-2 INFO systemd-udevd[6833]: Using default interface naming scheme 'v247'.

~~~
#### Which release branch to backport (provide reason below if selected)



- [ x] 202012
- [ x] 202106

#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
